e2e/cypress/support/ui_commands.js

```diff
@@ -443,3 +443,70 @@ Cypress.Commands.add('fileUpload', (targetInput, fileName = 'mattermost-icon.png
         );
     });
 });
+
+/**
+ * Upload a file on target input in binary format -
+ * @param {String} targetInput - #LocatorID
+ * @param {String} fileName - Filename to upload from the fixture Ex: drawPlugin-binary.tar
+ * @param {String} fileType - application/gzip
+ */
+Cypress.Commands.add('uploadFile', {prevSubject: true}, (targetInput, fileName, fileType) => {
+    cy.log('Upload process started .FileName:' + fileName);
+    cy.fixture(fileName, 'binary').then((content) => {
+        return Cypress.Blob.binaryStringToBlob(content, fileType).then((blob) => {
+            const el = targetInput[0];
+            cy.log('el:' + el);
+            const testFile = new File([blob], fileName, {type: fileType});
+            const dataTransfer = new DataTransfer();
+            dataTransfer.items.add(testFile);
+            el.files = dataTransfer.files;
+            cy.wrap(targetInput).trigger('change', {force: true});
+        });
+    });
+});
+
+/**
+ * Navigate to system console-PluginManagement from account settings
+ */
+Cypress.Commands.add('systemConsolePluginManagement', () => {
+    cy.get('#lhsHeader').should('be.visible').within(() => {
+        // # Click hamburger main menu
+        cy.get('#sidebarHeaderDropdownButton').click();
+
+        // * Dropdown menu should be visible
+        cy.get('.dropdown-menu').should('be.visible').within(() => {
+            // * Plugin Marketplace button should be visible then click
+            cy.get('#systemConsole').should('be.visible').click();
+        });
+    });
+
+    //Search for plugin management in filter container
+    cy.get('li.filter-container').find('input#adminSidebarFilter.filter').
+        wait(TIMEOUTS.TINY).should('be.visible').type('plugin Management').click();
+});
+
+/**
+ * Method to enable/Disable/Remove draw plugin
+ * @param {String} status - Action to be performed on draw plugin help text [Remove/Enable/Disable]
+ * @param {String} fileName - Filename to upload from the fixture
+ * @param {String} fileType - formation of the file - 'application/gzip'
+ */
+Cypress.Commands.add('drawpluginConfiguration', (status, fileName, fileType) => {
+    //Navigate to system console - Plugin Management - Enable Draw plugin
+    cy.wait(TIMEOUTS.TINY);
+    switch (status) {
+    case 'Remove':
+        cy.get('span > a[data-plugin-id="com.mattermost.draw-plugin"]').findByText('Remove').click().wait(TIMEOUTS.TINY);
+        cy.get('#confirmModalButton').should('be.visible').wait(TIMEOUTS.TINY).click();
+        break;
+    case 'Enable':
+        cy.get('input[type=file]').uploadFile(fileName, fileType).wait(TIMEOUTS.TINY);
+        cy.get('#uploadPlugin').should('be.visible').click().wait(TIMEOUTS.TINY);
+        cy.wait(TIMEOUTS.TINY);
+        cy.get('#pluginConfigSettings > a[data-plugin-id="com.mattermost.draw-plugin"]').findByText('Enable').click();
```

Comment:
This failed on "Error: Found multiple elements with the text: Enable" where there are 2 or more plugins installed. `findByText('Enable')` should be scoped by using `within`. With my comment on adding `data-testid`, I suggest this to be:
```suggestion
        cy.findByTestId('com.mattermost.draw-plugin').should('be.visible').within(() => {
            cy.findByText('Enable').click();
            cy.findByText('This plugin is running.').should('be.visible');
        });
```